test/libyul/yulOptimizerTests/redundantAssignEliminator/multivar.yul

```diff
@@ -0,0 +1,15 @@
+{
+    let a := 2
+    a := 7
+    let b := 8
+    b := a
+    a := b
+}
+// ----
+// redundantAssignEliminator
+// {
+//     let a := 2
```

Comment:
It might, but then this step would be more complicated. If you run `ssaTransform` before this step, `let a := 2` will be transformed to `let a_1 := 2 \n a := a_1` and if `a_1` is unused, both will be removed if you run `redundantAssignEliminator` and `unusedPruner`.